### DISCUSSION TITLE
Feedback

### DISCUSSION THREAD

**Discussion by pdimu** [PranshulGG/WeatherMaster#19] [General]
I wanted to give you feedback on this amazing app and didn't want to make a whole issue about it (also this is not an issue so discussions is better)

First of all, the way it clones the Google Weather shortcut app is so cool and I love that it also adds more features 

Second of all, the update cycle is very often (I still can't believe that you made this whole thing in just 3 months). Like, you update the app daily and sometimes the "distance" between two versions is just a few hours

**Third** of all, although the basics are there, most of the advanced features are still in development or not even done _yet_. I'll always be here to give you feature ideas when I have

**Comment by pdimu** [comment]
We got featured on the Material Apps List!
https://t.me/MaterialYouAppsDiscussion/279

**Reply by PranshulGG** [reply]
Whoa I didn’t expect that

**Reply by pdimu** [reply]
It's not on the list in the repo but I think it'll be added soon (I hope)

**Reply by pdimu** [reply]
I'll add it myself I guess

**Comment by bbrockie** [comment]
Great app! Is it possible to disable the auto update of it though?

**Comment by PranshulGG** [comment]
> Great app! Is it possible to disable the auto update of it though? 

It doesn't automatically update.

**Comment by V-Detach** [comment]
The dynamic colors setting (Android 12+) does not work on my Android 12 (tablet)
